### PR TITLE
refactor: icon to icons for homepage and card compompents

### PR DESCRIPTION
**Pull request by pkdotson**
### SUMMARY
This pr migrates the old icon to icons components for the chart card and home page card components.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Screen Shot 2021-07-09 at 2 45 39 PM

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/views/CRUD/welcome/ActivityTable.tsx [hunk 1]
```diff
@@ -116,16 +118,16 @@ const getEntityTitle = (entity: ActivityObject) => {
   return entity.item_title || UNTITLED;
 };
 
-const getEntityIconName = (entity: ActivityObject) => {
-  if ('sql' in entity) return 'sql';
+const getEntityIcons = (entity: ActivityObject) => {
```
Comment (suddjian) [inline superset-frontend/src/views/CRUD/welcome/ActivityTable.tsx:121]: I think this being plural is confusing. Seems to indicate we are returning multiple icons.

Comment (michael-s-molina) [reply superset-frontend/src/views/CRUD/welcome/ActivityTable.tsx:121]: Agree.

**Review by michael-s-molina — APPROVED** [review]
Approved with @suddjian suggestion.

**Review by geido — APPROVED** [review]
LGTM!

**Pull request merged**